koswat/cost_report/infrastructure/infrastructure_location_profile_cost_report.py

```diff
@@ -25,15 +24,22 @@ class InfrastructureLocationProfileCostReport(CostReportProtocol):
     infrastructure_location_costs: InfrastructureLocationCosts
 
     @property
-    def location(self) -> Point:
+    def location(self) -> PointSurroundings:
         if not self.infrastructure_location_costs:
-            return None
-        return self.infrastructure_location_costs.location.location
+            return PointSurroundings()
+        return self.infrastructure_location_costs.location
```

Comment:
Why is this change required? I do no believe you require such information anymore (you do have the `reinforced_profile` and `infrastructure` define in this class. 
Instead you should just return the coordinates related to this report. If there is not such point then just return `None`.

Reply:
We need the `section` in the report, which is not in `self.infrastructure_location_costs.location.location`.
Returning `None` is still fine. Adjusted.